.goreleaser.yml

```diff
@@ -0,0 +1,39 @@
+# Release automation via GoReleaser (goreleaser.com)
+# Requires a valid GITHUB_TOKEN envar prior to running `goreleaser`
+# See https://goreleaser.com/environment/ for more info
+---
+release:
+  github:
+    owner: golang
+    name: mock
+
+builds:
+  - binary: mockgen
+    goos:
+      - darwin
+      - windows
+      - linux
+    goarch:
+      - amd64
+      - 386
+    env:
+      - CGO_ENABLED=0
+    main: ./mockgen/
+archive:
+  format: tar.gz
+  wrap_in_directory: true
+  name_template: '{{ .Binary }}-{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
```

Comment:
What does `.Arm` correspond to and why do we want the `v` suffix?

Reply:
`.Arm` corresponds to the value of GOARM as specified on [this page](https://github.com/golang/go/wiki/GoArm) and is used to distinguish between the different versions. If you don't wish to support ARM, this can be removed.

Reply:
Ah, perfect! I was missing those docs. Thanks for the clarification and I see no reason to exclude ARM.